pkg/trait/logging_test.go

```diff
@@ -212,3 +213,31 @@ func TestJsonLoggingTrait(t *testing.T) {
 	assert.True(t, logFormatIsNotDefault)
 	assert.NotEmpty(t, env.ExecutedTraits)
 }
+
+func TestLoggingCategory(t *testing.T) {
```

Comment:
Make another test to include 2 variables and make sure the 2 different packages are correctly transformed.